### DISCUSSION TITLE
v2.4.1

### DISCUSSION THREAD

**Discussion by mashb1t** [mashb1t/Fooocus#34] [Announcements]
## What's Changed (synched from upstream)
* fix: adjust clip skip default value from 1 to 2 by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/3011
* fix: add type check for undefined, use fallback when no translation for aspect ratios was given by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/3025
* feat: build docker image tagged "edge" on push to main branch by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/3026

**Full Changelog**: https://github.com/lllyasviel/Fooocus/compare/v2.4.0...v2.4.1

<hr /><em>This discussion was created from the release <a href='https://github.com/mashb1t/Fooocus/releases/tag/v2.4.1'>v2.4.1</a>.</em>

**Comment by soyv4** [comment]
Good afternoon! ,tell me how to make sure that the Translate Prompts checkmark is enabled by default at the start of Fooocus, I change the file webui.py line 357 value=False to value=True , this works , but when updating Fooocus or checking this value is set to True. How to implement this or in which file to correct it, tell me?

**Reply by mashb1t** [reply]
This is currently the best solution until a config option / argument is added.
If you want to keep this line persostent on updates, you have to manually update Fooocus (either your own fork or manual pull) and not use entry_with_update.py but launch.py as entrypoint. Automated updates are ghen not available anymore.
